docs: Update FAQ about known types (#6008) (#20929)
argoproj/argo-cd: docs/faq.md

You can change the setting by updating the `timeout.reconciliation` value and the `timeout.reconciliation.jitter` in the [argocd-cm](https://github.com/argoproj/argo-cd/blob/2d6ce088acd4fb29271ffb6f6023dbb27594d59b/docs/operator-manual/argocd-cm.yaml#L279-L282) config map. If there are any Git changes, Argo CD will only update applications with the [auto-sync setting](user-guide/auto_sync.md) enabled. If you set it to `0` then Argo CD will stop polling Git repositories automatically and you can only use alternative methods such as [webhooks](operator-manual/webhook.md) and/or manual syncs for deploying applications.

> ## Why is my ArgoCD application `Out Of Sync` and keep syncing when there are no actual changes in resource limits or other fields with unit values?

Reviewer: ```suggestion
## Why does my ArgoCD application keep syncing when there are no actual changes in resource limits or other fields with unit values?
```

how about this?

Author: I'd keep about out of sync, to also cover people who have auto sync disabled and still wondering about the issue.

Reviewer 2: Or this one 

```suggestion
## Why is my ArgoCD application `Out Of Sync` when there are no actual changes to the resource limits (or other fields with unit values)?
```

Author: Done

> Kubernetes has normalized your resource limits when they are applied, and then Argo CD has then compared the version in

Reviewer: ```suggestion
Kubernetes has normalized your resource limits when they are applied, and then Argo CD compared the version in
```

> your generated manifests to the normalized one is Kubernetes - they may not match.

Reviewer: ```suggestion
your generated manifests from git to the normalized one in Kubernetes cluster - they may not match.
```
